bindgen/src/generator.ts

```diff
@@ -16,12 +16,13 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import { Spec } from "./spec";
+import { OptInSpec, Spec } from "./spec";
```

Comment:
My main gripe is that "opt-in" is two words 🙈

Reply:
Thanks for the input, I'll leave the conversation unresolved for now to see if others want to weigh in on this 👍

Reply:
I have a slight preference for "includeList"/"isIncluded", but not enough to overrule the patch author. Just stating my preference in case it comes to a vote.